conda_smithy/feedstock_io.py

```diff
@@ -7,22 +7,23 @@
 def get_repo(path, search_parent_directories=True):
     repo = None
     try:
-        import git
-
-        repo = git.Repo(
-            path, search_parent_directories=search_parent_directories
-        )
+        import pygit2
+
+        if search_parent_directories:
+            path = pygit2.discover_repository(path)
+        if path is not None:
+            repo = pygit2.Repository(
+                path, pygit2.enums.RepositoryOpenFlag.NO_SEARCH
+            )
```

Comment:
FWICS we never call it with `search_parent_directories=False`. If we were to remove that kwarg, we could just remove `NO_SEARCH` (or we could change flags based on the arg value — but I've figured doing it more explicitly is more readable).

Reply:
Done that anyway, since tests were failing — I guess this would be more cross-version compatible.